infra/base-images/base-builder/indexer/index_build.py

```diff
@@ -120,10 +121,10 @@ def _save_dir(path: Path,
               continue
 
             file = Path(root, file)
-            if exclude_build_artifacts and _is_elf(file):
+            if exclude_build_artifacts and _is_excludable_elf(file):
               continue
 
-            if only_include_target and _is_elf(file):
+            if only_include_target and _is_excludable_elf(file):
               # Skip ELF files that aren't the relevant target (unless it's a
               # shared library).
               if file.name != only_include_target and file.suffix != '.so':
```

Comment:
Should we change this to `'.so' in file.name` ?

Reply:
Sure. I wonder if better to just drop files that have LLVMFuzzerTestOneInput and aren't .o